playbooks/09-compliance.yml

```diff
@@ -0,0 +1,22 @@
+- name: Run compliance scan for controllers
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Run compliance scan for controllers
+      ansible.builtin.import_role:
+        name: compliance
+      vars:
+        cifmw_compliance_podman_username: "{{ cifmw_registry_token.credentials.username }}"
+        cifmw_compliance_podman_password: "{{ cifmw_registry_token.credentials.password }}"
+      when: cifmw_run_controller_compliance_scans | default('false') | bool
```

Comment:
*maybe* this parameter should match the actual action, `cifmw_run_operators_compliance_scans` ?